<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tclist</class>
 <widget class="QMainWindow" name="tclist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>991</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 242, 242);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>401</width>
       <height>541</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(218, 218, 218);
border-radius: 10px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QListWidget" name="teacherListWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>221</width>
        <height>361</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <underline>false</underline>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
  border-bottom: 1px solid #ccc; /* Border between rows */
 border-left: 1px solid #ccc; /* Border between rows */
 border-top: 1px solid #ccc; /* Border between rows */
    padding: 5px; /* Padding for each item */
    text-align: center;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>true</bool>
      </property>
      <property name="uniformItemSizes">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#000000;</string>
      </property>
      <property name="text">
       <string>Course</string>
      </property>
     </widget>
     <widget class="QListWidget" name="courseListWidget">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>120</y>
        <width>151</width>
        <height>361</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <underline>false</underline>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
  border-bottom: 1px solid #ccc; /* Border between rows */
 border-right: 1px solid #ccc; /* Border between rows */
 border-top: 1px solid #ccc; 
    padding: 5px; /* Padding for each item */
    text-align: center;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>true</bool>
      </property>
      <property name="uniformItemSizes">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#000000;</string>
      </property>
      <property name="text">
       <string>Teacher</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>461</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(55, 122, 183);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Product Sans Black</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>FACULTY LIST</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="logoutButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>500</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #337ab7;  /* Button background color (blue) */
    color: #ffffff;             /* Text color (white) */
	border-radius: 3px;
}
QPushButton:hover{
	background-color: #285e8e;
}
</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>290</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>0</y>
       <width>461</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(55, 122, 183);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>0</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>1</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Images.qrc">:/new/prefix1/wvsu_logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>8</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>WEST VISAYAS STATE UNIVERSITY
MAIN CAMPUS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>8</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Lapaz, Iloilo CIty</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Product Sans Black</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>DASHBOARD</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>118</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>16</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <zorder>label_7</zorder>
     <zorder>label_8</zorder>
     <zorder>label_9</zorder>
     <zorder>line_2</zorder>
     <zorder>line_3</zorder>
     <zorder>label_5</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>424</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
